src/components/modals/DeleteDerivationGroupModal.svelte

```diff
@@ -0,0 +1,76 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+  import type { DerivationGroup } from '../../types/external-source';
+  import Modal from './Modal.svelte';
+  import ModalContent from './ModalContent.svelte';
+  import ModalFooter from './ModalFooter.svelte';
+  import ModalHeader from './ModalHeader.svelte';
+
+  export let height: number = 150;
+  export let derivationGroup: DerivationGroup;
+  export let width: number = 380;
+
+  const dispatch = createEventDispatcher<{
+    close: void;
+    confirm: void;
+  }>();
+
+  function onKeydown(event: KeyboardEvent) {
+    const { key } = event;
+    if (key === 'Enter') {
+      event.preventDefault();
+      dispatch('confirm');
+    }
+  }
+</script>
+
+<svelte:window on:keydown={onKeydown} />
+
+<Modal {height} {width}>
+  <ModalHeader on:close>
+    {#if derivationGroup.sources.size > 0}
+      Derivation Group Cannot Be Deleted
+    {:else}
+      Delete External Source
+    {/if}
+  </ModalHeader>
+  <ModalContent>
+    {#if derivationGroup.sources.size > 0}
+      <span>
+        This Derivation Group still contains the following sources which must be deleted first:
+        <hr style="border: 0px" />
+        {#each derivationGroup.sources as source}
+          <div class="modal-content" style:padding-left="20px">
+            <i>
+              {source[0]}
+            </i>
+          </div>
+        {/each}
+      </span>
+    {:else}
+      <span class="modal-content">
+        Are you sure you want to delete "{derivationGroup.name}"?
+        <br />
+        <i>What is done cannot be undone.</i>
```

Comment:
Can you say "This action cannot be undone." here and in the other instances to match the language we use elsewhere in the app?